utils/cautils/offline.go

```diff
@@ -45,23 +46,23 @@ func NewOfflineCA(configFile string) (*OfflineCA, error) {
 		return nil, err
 	}
 
-	var config authority.Config
-	if err = json.Unmarshal(b, &config); err != nil {
+	var cfg config.Config
+	if err = json.Unmarshal(b, &cfg); err != nil {
 		return nil, errors.Wrapf(err, "error reading %s", configFile)
 	}
 
-	if config.AuthorityConfig == nil || len(config.AuthorityConfig.Provisioners) == 0 {
+	if cfg.AuthorityConfig == nil || len(cfg.AuthorityConfig.Provisioners) == 0 {
 		return nil, errors.Errorf("error parsing %s: no provisioners found", configFile)
 	}
 
-	auth, err := authority.New(&config)
+	auth, err := authority.New(&cfg)
```

Comment:
We don't have offline support on the new commands, at some point we should add it.